bot/cogs/watchchannels/watchchannel.py

```diff
@@ -0,0 +1,359 @@
+            except (errors.Forbidden, errors.NotFound):
+                e = Embed(
+                    description=":x: **This message contained an attachment, but it could not be retrieved**",
+                    color=Color.red()
+                )
+                await self.webhook_send(
+                    embed=e,
+                    username=msg.author.display_name,
+                    avatar_url=msg.author.avatar_url
+                )
+            except discord.HTTPException as exc:
+                self.log.exception(
+                    f"Failed to send an attachment to {self.__class__.__name__} webhook",
+                    exc_info=exc
+                )
+
+        self.message_history[2] += 1
+
+    async def send_header(self, msg) -> None:
+        """Sends a header embed to the WatchChannel"""
+        user_id = msg.author.id
+
+        guild = self.bot.get_guild(GuildConfig.id)
+        actor = guild.get_member(self.watched_users[user_id]['actor'])
+        actor = actor.display_name if actor else self.watched_users[user_id]['actor']
+
+        inserted_at = self.watched_users[user_id]['inserted_at']
+        time_delta = self._get_time_delta(inserted_at)
+
+        reason = self.watched_users[user_id]['reason']
+
+        embed = Embed(description=(
+            f"{msg.author.mention} in [#{msg.channel.name}]({msg.jump_url})\n"
+        ))
+        embed.set_footer(text=(
+            f"Added {time_delta} by {actor} | "
+            f"Reason: {reason}"
+        ))
+        await self.webhook_send(
+            embed=embed,
+            username=msg.author.display_name,
+            avatar_url=msg.author.avatar_url
+        )
+
+    async def list_watched_users(self, ctx: Context, update_cache: bool = True) -> None:
+        """
+        Gives an overview of the watched user list for this channel.
+
+        The optional kwarg `update_cache` specifies whether the cache should
+        be refreshed by polling the API.
+        """
+        if update_cache:
+            if not await self.fetch_user_cache():
+                e = Embed(
+                    description=f":x: **Failed to update {self.__class__.__name__} user cache, serving from cache**",
+                    color=Color.red()
+                )
+                await ctx.send(embed=e)
+                update_cache = False
+
+        lines = []
+        for user_id, user_data in self.watched_users.items():
+            inserted_at = user_data['inserted_at']
+            time_delta = self._get_time_delta(inserted_at)
+            lines.append(f"• <@{user_id}> (added {time_delta})")
+
+        await LinePaginator.paginate(
+            lines or ("There's nothing here yet.",),
+            ctx,
+            Embed(
+                title=f"{self.__class__.__name__} watched users ({'updated' if update_cache else 'cached'})",
+                color=Color.blue()
+            ),
+            empty=False
```

Comment:
OK, so this follows our style but it's kinda ugly since it's so disproportionate. Do you think this looks better?

```py
        lines = lines or ("There's nothing here yet.",)
        embed = Embed(
            title=f"{self.__class__.__name__} watched users ({'updated' if update_cache else 'cached'})",
            color=Color.blue()
        )
        await LinePaginator.paginate(lines, ctx, embed empty=False)
```

Preferably the embed's title would also be under 100 columns.

Reply:
I've changed to formatting but not the line length. As it stands now, there are a lot of other lines that go over a `100` as well. Since there's been some discussion of using `black` with a line limit of `120`, I've left this as it is for now.

If we are going to use `black`, it's going to produce a lot of lines with near 120 columns.

Reply:
A while ago it was confirmed we wanted to target 100 columns but just haven't done it yet for certain repos. But that was like a month ago and I guess we change our minds. I don't know what's going on with black really. Let's just worry about that later.

Reply:
All new code should adhere to a line length of 100.

`black` was a thought experiment

Reply:
Disregard, I can't read properly. The current stance is a line length of 100 for new projects.